docs/mkdocs.yml

```diff
@@ -1,11 +1,12 @@
 site_name: Conductor
 repo_url: https://github.com/Netflix/conductor
+edit_uri: edit/master/docs/docs/
```

Comment:
Does this points to master branch? New edit branches should merge into `dev` first.

Reply:
You are right. I fixed it 😄